mkdocs.yml

```diff
@@ -101,6 +101,7 @@ nav:
       - Documentation Guide: developer-guide/documentation.md
       - Local Testing Guide: developer-guide/testing.md
       - CI Testing Guide: ci/ci.md
+      - Web Design Guide: developer-guide/web-design.md
```

Comment:
hmm we should make this the documentation.md guide which is missing atm? L101 that's what I had in mind at least. I guess we could/should merge them?

Reply:
the `documentation.md` doesn't exist today and that should be `web-design.md`;  I'd prefer calling it documentation at the end of the day cc @girishmg - we should add other sections like how each PR should have a docs changes associated with it etc to that but more or less all this should go into documentation.md